<ui version="4.0" >
 <class>VCardDialogClass</class>
 <widget class="QDialog" name="VCardDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>VCardDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="twtVCard" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral" >
      <property name="autoFillBackground" >
       <bool>true</bool>
      </property>
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="lotGeneral" >
       <property name="margin" >
        <number>3</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="lblFullName" >
         <property name="text" >
          <string>Full name</string>
         </property>
         <property name="buddy" >
          <cstring>lneFullName</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="lneFullName" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="lblFirstName" >
         <property name="text" >
          <string>First name</string>
         </property>
         <property name="buddy" >
          <cstring>lneFirstName</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="lneFirstName" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="lblMiddleName" >
         <property name="text" >
          <string>Middle name</string>
         </property>
         <property name="buddy" >
          <cstring>lneMiddleName</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="lneMiddleName" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="lblLastName" >
         <property name="text" >
          <string>Last name</string>
         </property>
         <property name="buddy" >
          <cstring>lneLastName</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="lneLastName" />
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="lblNickName" >
         <property name="text" >
          <string>Nick name</string>
         </property>
         <property name="buddy" >
          <cstring>lneNickName</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QLineEdit" name="lneNickName" />
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="lblJabberId" >
         <property name="text" >
          <string>Jabber Id</string>
         </property>
         <property name="buddy" >
          <cstring>lneJabberId</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QLineEdit" name="lneJabberId" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPersonal" >
      <property name="autoFillBackground" >
       <bool>false</bool>
      </property>
      <attribute name="title" >
       <string>Personal</string>
      </attribute>
      <layout class="QFormLayout" name="lotPersonal" >
       <property name="margin" >
        <number>3</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="lblBirthday" >
         <property name="text" >
          <string>Birthday</string>
         </property>
         <property name="buddy" >
          <cstring>dedBirthday</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <layout class="QHBoxLayout" name="lotPersonalBirthday" >
         <item>
          <widget class="QDateEdit" name="dedBirthday" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="dateTime" >
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>1900</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="minimumDateTime" >
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>1900</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="displayFormat" >
            <string>dd.MMMM.yyyy</string>
           </property>
           <property name="calendarPopup" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblGender" >
           <property name="text" >
            <string>Gender</string>
           </property>
           <property name="buddy" >
            <cstring>cmbGender</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbGender" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable" >
            <bool>true</bool>
           </property>
           <item>
            <property name="text" >
             <string>Male</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Female</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="lblMaritial" >
         <property name="text" >
          <string>Maritial status</string>
         </property>
         <property name="buddy" >
          <cstring>lneMaritial</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="lneMaritial" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="lblOcuupation" >
         <property name="text" >
          <string>Occupation</string>
         </property>
         <property name="buddy" >
          <cstring>lneTitle</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="lneTitle" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="lblCompany" >
         <property name="text" >
          <string>Company</string>
         </property>
         <property name="buddy" >
          <cstring>lneCompany</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="lneCompany" />
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="lblDepartment" >
         <property name="text" >
          <string>Department</string>
         </property>
         <property name="buddy" >
          <cstring>lneDepartment</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QLineEdit" name="lneDepartment" />
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="lblRole" >
         <property name="text" >
          <string>Role</string>
         </property>
         <property name="buddy" >
          <cstring>lneRole</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QLineEdit" name="lneRole" />
       </item>
       <item row="6" column="0" >
        <widget class="QLabel" name="lblHomePage" >
         <property name="text" >
          <string>Home page</string>
         </property>
         <property name="buddy" >
          <cstring>lneHomePage</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1" >
        <widget class="QLineEdit" name="lneHomePage" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdress" >
      <property name="autoFillBackground" >
       <bool>true</bool>
      </property>
      <attribute name="title" >
       <string>Adress</string>
      </attribute>
      <layout class="QHBoxLayout" name="lotAdress" >
       <property name="margin" >
        <number>3</number>
       </property>
       <item>
        <widget class="QGroupBox" name="grbHomeAdress" >
         <property name="title" >
          <string>Home</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="lotAdressHome" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="lblHomeStreet" >
            <property name="text" >
             <string>Street</string>
            </property>
            <property name="buddy" >
             <cstring>lneHomeStreet</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="lneHomeStreet" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="lblHomeCity" >
            <property name="text" >
             <string>City</string>
            </property>
            <property name="buddy" >
             <cstring>lneHomeCity</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="lneHomeCity" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="lblHomeState" >
            <property name="text" >
             <string>State</string>
            </property>
            <property name="buddy" >
             <cstring>lneHomeState</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="lneHomeState" />
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="lblHomeZip" >
            <property name="text" >
             <string>Zip code</string>
            </property>
            <property name="buddy" >
             <cstring>lneHomeZip</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QLineEdit" name="lneHomeZip" />
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="lblHomeCountry" >
            <property name="text" >
             <string>Country</string>
            </property>
            <property name="buddy" >
             <cstring>lneHomeCountry</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QLineEdit" name="lneHomeCountry" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grbWorkAdress" >
         <property name="title" >
          <string>Work</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="lotAdressWork" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="lblWorkStreet" >
            <property name="text" >
             <string>Street</string>
            </property>
            <property name="buddy" >
             <cstring>lneWorkStreet</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="lneWorkStreet" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="lblWorkCity" >
            <property name="text" >
             <string>City</string>
            </property>
            <property name="buddy" >
             <cstring>lneWorkCity</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="lneWorkCity" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="lblWorkState" >
            <property name="text" >
             <string>State</string>
            </property>
            <property name="buddy" >
             <cstring>lneWorkState</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="lneWorkState" />
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="lblWorkZip" >
            <property name="text" >
             <string>Zip code</string>
            </property>
            <property name="buddy" >
             <cstring>lneWorkZip</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QLineEdit" name="lneWorkZip" />
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="lblWorkCountry" >
            <property name="text" >
             <string>Country</string>
            </property>
            <property name="buddy" >
             <cstring>lneWorkCountry</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QLineEdit" name="lneWorkCountry" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabContacts" >
      <property name="autoFillBackground" >
       <bool>true</bool>
      </property>
      <attribute name="title" >
       <string>Contacts</string>
      </attribute>
      <layout class="QHBoxLayout" name="lotContacts" >
       <property name="margin" >
        <number>3</number>
       </property>
       <item>
        <widget class="QGroupBox" name="grbEmail" >
         <property name="title" >
          <string>E-Mail</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="lotEmail" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="ltwEmails" />
          </item>
          <item>
           <layout class="QHBoxLayout" name="lotEmailButtons" >
            <item>
             <widget class="QToolButton" name="tlbEmailAdd" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tlbEmailDelete" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grbPhone" >
         <property name="title" >
          <string>Phone</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="lotPhone" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="ltwPhones" />
          </item>
          <item>
           <layout class="QHBoxLayout" name="lotPhoneButtons" >
            <item>
             <widget class="QToolButton" name="tlbPhoneAdd" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tlbPhoneDelete" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPhoto" >
      <property name="autoFillBackground" >
       <bool>true</bool>
      </property>
      <attribute name="title" >
       <string>Photo</string>
      </attribute>
      <layout class="QHBoxLayout" name="lotPhoto" >
       <property name="margin" >
        <number>3</number>
       </property>
       <item>
        <widget class="QGroupBox" name="grbPhoto" >
         <property name="title" >
          <string>Photo</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="lotPhotoImage" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lblPhoto" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="lotPhotoButtons" >
            <item>
             <widget class="QToolButton" name="tlbPhotoSave" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tlbPhotoLoad" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Load</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tlbPhotoClear" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grbLogo" >
         <property name="title" >
          <string>Logo</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="lotLogoImage" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lblLogo" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="lotLogoButtons" >
            <item>
             <widget class="QToolButton" name="tlbLogoSave" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tlbLogoLoad" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Load</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tlbLogoClear" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabComments" >
      <property name="autoFillBackground" >
       <bool>true</bool>
      </property>
      <attribute name="title" >
       <string>Comments</string>
      </attribute>
      <layout class="QVBoxLayout" name="lotComments" >
       <property name="margin" >
        <number>3</number>
       </property>
       <item>
        <widget class="QTextEdit" name="tedComments" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lotButtons" >
     <item>
      <widget class="QPushButton" name="pbtUpdate" >
       <property name="text" >
        <string>Reload</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtPublish" >
       <property name="text" >
        <string>Publish</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtClear" >
       <property name="text" >
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtClose" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>twtVCard</tabstop>
  <tabstop>lneFullName</tabstop>
  <tabstop>lneFirstName</tabstop>
  <tabstop>lneMiddleName</tabstop>
  <tabstop>lneLastName</tabstop>
  <tabstop>lneNickName</tabstop>
  <tabstop>lneJabberId</tabstop>
  <tabstop>dedBirthday</tabstop>
  <tabstop>cmbGender</tabstop>
  <tabstop>lneMaritial</tabstop>
  <tabstop>lneTitle</tabstop>
  <tabstop>lneDepartment</tabstop>
  <tabstop>lneCompany</tabstop>
  <tabstop>lneRole</tabstop>
  <tabstop>lneHomePage</tabstop>
  <tabstop>lneHomeStreet</tabstop>
  <tabstop>lneHomeCity</tabstop>
  <tabstop>lneHomeState</tabstop>
  <tabstop>lneHomeZip</tabstop>
  <tabstop>lneHomeCountry</tabstop>
  <tabstop>lneWorkStreet</tabstop>
  <tabstop>lneWorkCity</tabstop>
  <tabstop>lneWorkState</tabstop>
  <tabstop>lneWorkZip</tabstop>
  <tabstop>lneWorkCountry</tabstop>
  <tabstop>pbtClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
